src/controllers/product.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-underscore-dangle */
+const objectId = require('mongoose').Types.ObjectId;
+const UserModel = require('../models/user');
+const ProductModel = require('../models/product');
+
+const createProduct = async (req, res) => {
```

Comment:
naming can be addProduct maybe but not that important, it is just that create is a huge word imo :D